atomistics/workflows/evcurve/workflow.py

```diff
@@ -158,14 +158,7 @@ def generate_structures(self):
         Returns:
             (dict)
         """
-        strains = self.strains
-        if strains is None:
-            strains = np.linspace(
-                -self.vol_range,
-                self.vol_range,
-                int(self.num_points),
-            )
-        for strain in strains:
+        for strain in self._get_strains():
```

Comment:
Post property suggestion. 

```suggestion
        for strain in self.strains:
```